src/sections/Kanvas/kanvas_banner.js

```diff
@@ -158,16 +159,61 @@ const BannerSectionWrapper = styled.div`
             margin-top: -5rem;
             margin-right: 2rem;
             left: 0%;
-            min-width: 3rem;
+            min-width: 8rem;
             }
         border-radius: 0% 85% 0% 0% / 0% 40% 0% 0% ;
         }
+    @media screen and (max-width: 450px) {
+        .kanvas-logo {
+            margin-top: -5rem;
+            margin-right: 1rem;
+            left: -10%;
+            min-width: 7rem;
+            }
+        border-radius: 0% 85% 0% 0% / 0% 40% 0% 0% ;
+        }    
     @media screen and (max-width: 400px) {
         .kanvas-logo {
             margin-top: -5rem;
-            margin-right: 0;
-            left: -20%;
-            min-width: 3rem;
+            margin-right: 0rem;
+            left: -28%;
+            min-width: 6rem;
+            }
+        border-radius: 0% 85% 0% 0% / 0% 40% 0% 0% ;
+        }
+    @media screen and (max-width: 350px) {
+        .kanvas-logo {
+            margin-top: -5rem;
+            margin-right: 0rem;
+            left: -48%;
+            min-width: 5rem;
+            }
+        border-radius: 0% 85% 0% 0% / 0% 40% 0% 0% ;
+        }
+    @media screen and (max-width: 305px) {
+        .kanvas-logo {
+            margin-top: -5rem;
+            margin-right: 0rem;
+            left: -90%;
+            min-width: 5rem;
+            }
+        border-radius: 0% 85% 0% 0% / 0% 40% 0% 0% ;
+        }
+    @media screen and (max-width: 260px) {
+        .kanvas-logo {
+            margin-top: -5rem;
+            margin-right: 0rem;
+            left: -170%;
+            min-width: 5rem;
+            }
+        border-radius: 0% 85% 0% 0% / 0% 40% 0% 0% ;
+        }
+    @media screen and (max-width: 210px) {
```

Comment:
this overuse of media queries is not a good practice. This will become hard to maintain and when something changes around this component we again had to rewrite its styles. Use better css, like use flex, max width etc and make more dynamic rather than fixing on each screen width.

Reply:
Thanks for the suggestion, @sudhanshutech  I'll refactor the CSS using more dynamic techniques like flex and max-width instead of relying on media queries.